bond-no-treasury/deploy/001_deploy_Bond.ts

```diff
@@ -21,13 +29,8 @@ const deployBond: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   /// Oracle Address
   const hec_dai_oracle = '0x227ccdfebea1355e73f1936c8a23ebb5f0181b3c';
-
-  /// Initial Params
-  const fundRecipient = '';
-  const feeBps = 3000;
-  const feeRecipients = [''];
-  const weightBps = [10000];
-  const minimumPrice = 10000;
+  const hec_usdc_oracle = '';
+  const hec_usdt_oracle = '';
```

Comment:
@bejaxer please use 0x2561a0A9083BD002762cB9008b08baed00BeE28e for hec-usdt oracle (mainnet)